src/applications/burials-v2/components/NoFormPage.jsx

```diff
@@ -428,7 +428,12 @@ export const NoFormPage = () => {
     <div className="row vads-u-margin-bottom--4">
       <h1>Review burial benefits application</h1>
       <p>VA Form 21P-530</p>
-      <va-alert close-btn-aria-label="Close notification" status="info" visible>
+      <va-alert
+        close-btn-aria-label="Close notification"
+        status="info"
+        visible
+        uswds="false"
```

Comment:
What is the reason for not using a v3/`uswds="true"` alert here?

Reply:
@powellkerry this was part of our teams [temporary opting out of v3 components](https://github.com/department-of-veterans-affairs/va.gov-team/issues/75081) and should/will be revised

Reply:
@powellkerry we will not launch with uswds:false/true
I have created Issue [#79820 ](https://app.zenhub.com/workspaces/benefits-non-disability-645913c7d909c20011380ae8/issues/gh/department-of-veterans-affairs/va.gov-team/79820) to clean up this request.

We will not be making this change as part of this PR.